modules/installation-configure-proxy.adoc

```diff
@@ -202,18 +197,24 @@ baseDomain: my.domain.com
 proxy:
   httpProxy: http://<username>:<pswd>@<ip>:<port> <1>
   httpsProxy: https://<username>:<pswd>@<ip>:<port> <2>
+ifndef::aws[]
   noProxy: example.com <3>
+endif::aws[]
+ifdef::aws[]
+  noProxy: ec2.<region>.amazonaws.com,elasticloadbalancing.<region>.amazonaws.com,s3.<region>.amazonaws.com <3>
+endif::aws[]
 additionalTrustBundle: | <4>
     -----BEGIN CERTIFICATE-----
     <MY_TRUSTED_CA_CERT>
     -----END CERTIFICATE-----
-...
 ----
 <1> A proxy URL to use for creating HTTP connections outside the cluster. The
 URL scheme must be `http`.
 <2> A proxy URL to use for creating HTTPS connections outside the cluster.
-<3> A comma-separated list of destination domain names, IP addresses, or
-other network CIDRs to exclude from proxying. Preface a domain with `.` to match subdomains only. For example, `.y.com` matches `x.y.com`, but not `y.com`. Use `*` to bypass the proxy for all destinations.
+<3> A comma-separated list of destination domain names, IP addresses, or other network CIDRs to exclude from proxying. Preface a domain with `.` to match subdomains only. For example, `.y.com` matches `x.y.com`, but not `y.com`. Use `*` to bypass the proxy for all destinations.
+ifdef::aws[]
+If you have added the Amazon `EC2`,`Elastic Load Balancing`, and `S3` VPC endpoints to your VPC, you must add these endpoints to the `noProxy` field.
```

Comment:
> although this may be not recommended.

While scenario 4 is a possible solution, the documentation should only include recommended configurations. If this is a corner case (an "unusual configuration" as you put it) IMO we should leave Options 1-3. If we later get feedback from the field that customers want to configure scenario 4, it can always be added. Given the fact that the purpose of VPC endpoints is keep network traffic private to the respective service, and we provide an option without them (Option 2), I am left wondering why anyone would configure a route to allow "private" traffic over the internet via the proxy. WDYT?